base/BUILD.gn

```diff
@@ -2477,7 +2477,7 @@ buildflag_header("debugging_buildflags") {
 
   # TODO(crbug.com/1420774): Try and enable these checks on Android too.
   enable_commandline_sequence_checks =
-      (is_debug || dcheck_always_on) && !is_android
+      (is_debug || dcheck_always_on) && !is_android && !is_cobalt
```

Comment:
Is the `--single-process` not allowed when `enable_commandline_sequence_checks`?

Reply:
See the bug. On Linux, that triggers a thread checker assertion. On android, that is disabled here with the `!is_android` part of the expression.